addons/Popochiu/Editor/Importers/Aseprite/AnimationCreator.gd

```diff
@@ -0,0 +1,282 @@
+# This logic has been taken almost as-is from Vinicius Gerevini's
+# Aseprite Wizard plugin. Credits goes to him for the real magic.
+# See: https://godotengine.org/asset-library/asset/713
+tool
```

Comment:
@mapedorr unsure about this, but I guess it's ok to leave it as a tool script?

Reply:
If the plugin makes use of this script you have to leave the tool annotation.